src/NaiveBayesOnReviewsTest.java

```diff
@@ -0,0 +1,102 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by sol on 2015-11-01.
+ */
+public class NaiveBayesOnReviewsTest {
+
+    private static final String PATH_TO_RESOURCE = "/home/sol/code/machine-learning-classification/resources/amazon-balanced-6cats";
```

Comment:
I think this should be passed as a parameter when running the application, instead of being a constant. :)